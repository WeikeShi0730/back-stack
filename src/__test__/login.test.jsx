import { render } from "@testing-library/react";
import Login from "../pages/login/login.component";

describe("<Login />", () => {
  test("<Login />", () => {
    const { baseElement } = render(<Login />);
    expect(baseElement).toMatchSnapshot();
  });
});
