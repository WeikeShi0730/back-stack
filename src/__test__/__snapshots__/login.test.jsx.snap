// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> <Login /> 1`] = `
<body>
  <div>
    <div
      class="flex flex-col lg:flex-row lg:h-full w-full items-center justify-evenly"
    >
      <div
        class="m-5 p-5"
      >
        <div
          class="max-w-xs sm:max-w-sm md:max-w-md lg:max-w-lg m-auto mt-10 mb-4 bg-white rounded-lg border border-primaryBorder shadow-default py-8 px-10"
        >
          <h1
            class="text-base lg:text-lg font-light text-primary mt-4 mb-12 text-center"
          >
            Don't have an account? Sign up üîê
          </h1>
          <form>
            <div>
              <label
                class="text-sm md:text-base"
              >
                Username
              </label>
              <input
                class="text-xs md:text-md w-full p-2 text-primary border rounded-md outline-none transition duration-150 ease-in-out mb-4"
                maxlength="10"
                name="displayName"
                placeholder="Your username"
                required=""
                type="text"
              />
            </div>
            <div>
              <label
                class="text-sm md:text-base"
              >
                Email
              </label>
              <input
                class="text-xs md:text-md w-full p-2 text-primary border rounded-md outline-none transition duration-150 ease-in-out mb-4"
                name="email"
                placeholder="Your email"
                required=""
                type="email"
              />
            </div>
            <div>
              <label
                class="text-sm md:text-base"
              >
                Password
              </label>
              <input
                autocomplete="off"
                class="text-xs md:text-md w-full p-2 text-primary border rounded-md outline-none transition duration-150 ease-in-out mb-4"
                minlength="6"
                name="password"
                placeholder="Your password"
                required=""
                type="password"
              />
            </div>
            <div
              class="flex justify-center items-center mt-6"
            >
              <button
                class="text-xs md:text-sm bg-gray-800 py-2 px-4 text-white rounded border focus:outline-none font-light"
              >
                Sign up
              </button>
            </div>
          </form>
        </div>
      </div>
      <div
        class="m-5 p-5"
      >
        <div
          class="max-w-xs sm:max-w-sm md:max-w-md lg:max-w-lg m-auto mt-10 mb-4 bg-white rounded-lg border border-primaryBorder shadow-default py-8 px-10"
        >
          <div
            class="text-base lg:text-lg font-light text-primary mt-4 mb-12 text-center"
          >
            Alrady have an account? Sign in üîê
          </div>
          <form>
            <div>
              <label
                class="text-sm md:text-base"
              >
                Email
              </label>
              <input
                class="w-full p-2 text-xs md:text-md text-primary border rounded-md outline-none transition duration-150 ease-in-out mb-4"
                name="email"
                placeholder="Your email"
                required=""
                type="email"
              />
            </div>
            <div>
              <label
                class="text-sm md:text-base"
              >
                Password
              </label>
              <input
                autocomplete="off"
                class="w-full p-2 text-xs md:text-md text-primary border rounded-md outline-none transition duration-150 ease-in-out mb-4"
                name="password"
                placeholder="Your password"
                required=""
                type="password"
              />
            </div>
            <div
              class="flex justify-center items-center mt-6"
            >
              <button
                class="text-xs md:text-sm bg-gray-800 py-2 px-4 text-white rounded border focus:outline-none font-light"
                name="signInWithEmail"
                type="submit"
              >
                Sign In
              </button>
            </div>
            <div
              class="flex justify-center items-center mt-6"
            >
              <button
                class="text-xs md:text-sm bg-blue-500 py-2 px-4 text-white rounded border focus:outline-none focus:bg-gray-550 font-light"
                name="signInWithGoogle"
                type="submit"
              >
                Sign in with Google
              </button>
            </div>
          </form>
          <div>
            <button
              class="font-light"
            >
              Forgot password?
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
